### PR TITLE
feat: add option to display video captions

### PR DESCRIPTION
**Pull request by koetzsch**

### CODE REVIEW

File: src/editor/attributes.js [hunk 1]
```diff
@@ -30,4 +30,8 @@ export const attributes = {
         type: 'string',
         default: '',
     },
+    ccLangPref: {
+        type: 'array',
```
Comment (AndreasMaros) [inline src/editor/attributes.js:34]: Isn't this a single string value rather than an array?

File: src/editor/edit.js [hunk 2]
```diff
@@ -127,6 +130,14 @@ const VideoEditSettings = (props) => {
                         buttonChangeLabel="Change thumbnail image"
                         {...props}
                     />
+                    <SelectControl
+                      label={__('Caption Language Preference', 'ghwp')}
+                      value={ccLangPref}
+                      options={ccLangPrefOptions}
+                      onChange={(ccLangPref) => {
+                          setAttributes({ ccLangPref }); // @TODO: Add functionality to save this value
```
Comment (AndreasMaros) [inline src/editor/edit.js:138]: I don't understand this TODO

File: src/editor/attributes.js (hunk 1 above)
Comment (koetzsch) [reply src/editor/attributes.js]: @AndreasMaros : That's correct. I was confused about the language object.

File: src/editor/edit.js (hunk 2 above)
Comment (koetzsch) [reply src/editor/edit.js]: @AndreasMaros : That referred to the question in Slack. I thought the setAttributes function was not yet available here.

**Pull request closed without merging**